packages/cozy-stack-client/src/DocumentCollection.js

```diff
@@ -35,6 +35,7 @@ class DocumentCollection {
     this.doctype = doctype
     this.stackClient = stackClient
     this.indexes = {}
+    this.endpoint = '/data/'
```

Comment:
`this.endpoint = '/data/${this.doctype}/'` no ?

Reply:
Wops, right... Tests was missing here, I added some